src/pages/DistributePage/AllocationsTable.tsx

```diff
@@ -0,0 +1,261 @@
+import { useMemo, useState } from 'react';
+
+import { makeStyles } from '@material-ui/core';
+
+import { ApeAvatar, StaticTable } from 'components';
+import { shortenAddress } from 'utils';
+
+import { IUser, ITableColumn } from 'types';
+
+type AllocationTableProps = {
+  users: IUser[];
+};
+
+/**
+ * Component that displays a list of allocations.
+ * @param users IUser[]
+ * @returns
+ */
+const AllocationTable = ({ users }: AllocationTableProps) => {
+  const classes = useStyles();
+  const [keyword, setKeyword] = useState<string>('');
+  const onChangeKeyword = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setKeyword(event.target.value);
+  };
+  const filterUser = useMemo(
+    () => (u: IUser) => {
+      const r = new RegExp(keyword, 'i');
+      return r.test(u.name) || r.test(u.address);
+    },
+    [keyword]
+  );
+
+  const userColumns = useMemo(
+    () =>
+      [
+        {
+          label: 'Name',
+          accessor: 'name',
+          render: function UserName(u: IUser) {
+            return (
+              <div className={classes.avatarCell}>
+                <ApeAvatar user={u} className={classes.avatar} />
+                <span>{u.name}</span>
+              </div>
+            );
+          },
+          wide: true,
+          leftAlign: true,
+        },
+        {
+          label: 'ETH Wallet',
+          accessor: 'address',
+          render: (u: IUser) => shortenAddress(u.address),
+        },
+        {
+          label: 'Give Received',
+          render: () => '195 GIVE',
+        },
+        {
+          label: '# of Contributor Gitfing',
+          render: () => '22',
+        },
+        {
+          label: '% of Epoch',
+          render: () => '2.5%',
+        },
+        {
+          label: 'Vault Funds Allocated',
+          render: () => '1680 USDC',
+        },
+      ] as ITableColumn[],
+    []
+  );
+
+  return (
+    <>
+      <div className={classes.userActionBar}>
```

Comment:
For the `div` can we change to use Box from stitches and apply the styles with the stitches theme config?

Reply:
Done. Component does need a full rewrite of the static table component to migrate it permanently.